### PR TITLE
DevTools 105 > Perf insights > LCP badge

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/new-in-devtools-105/#lcp

Merge on Aug 30

### CODE REVIEW

File: site/en/docs/devtools/performance-insights/index.md [hunk 1]
```diff
@@ -129,6 +129,23 @@ Hover over the insights on the **Timeline** to learn more about the metrics.
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/23onsIV7BvDJpMI4mnMZ.mp4", controls="true", muted="true", class="screenshot" %}
 
+## Discover delays of the largest contentful paint {: #largest-contentful-paint }
+
+[Largest Contentful Paint (LCP)](https://web.dev/lcp/) is one of the [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) metrics that reports the render time of the largest image or text block visible within the viewport, relative to when the page first started loading.
```
Comment (rachelandrew) [inline site/en/docs/devtools/performance-insights/index.md:134]:
```suggestion
[Largest Contentful Paint (LCP)](https://web.dev/lcp/) is one of the [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) metrics. It reports the render time of the largest image or text block visible within the viewport, relative to when the page first started loading.
```

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/performance-insights/index.md [hunk 2]
```diff
@@ -129,6 +129,23 @@ Hover over the insights on the **Timeline** to learn more about the metrics.
...
+
+{% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/elqsdYqQEefWJbUM2qMO.svg", alt="LCP thresholds.", width="800", height="212" %}
+
+A [good LCP score](https://web.dev/lcp/#what-is-a-good-lcp-score?) is 2.5 seconds or less.
+
+If the largest contentful paint on your page takes longer to render, in the timeline, you will see the LCP badge with a yellow square or red triangle.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ftLjLtpo9Lki9JGbhHpB.png", alt="The LCP badge.", width="800", height="580" %}
+
+Click the badge to open the **Details** pane. In the pane, you can discover potential causes for the delays and suggestions on how to fix them.
```
Comment (jecfish) [inline site/en/docs/devtools/performance-insights/index.md:144]: There are two ways to open the LCP in Details pane, another way is to click on the LCP in the Insights pane.

**Review by jecfish — COMMENTED** [review]
LGTM with one more update

**Review by jecfish — APPROVED** [review]
LGTm with one more minor change

**Pull request merged**